diff -r -u -N a/usr/share/X11/xkb/rules/evdev b/usr/share/X11/xkb/rules/evdev
--- a/usr/share/X11/xkb/rules/evdev	2020-02-03 21:55:43.000000000 +0100
+++ b/usr/share/X11/xkb/rules/evdev	2020-02-22 09:20:21.749291098 +0100
@@ -1142,6 +1142,7 @@
   lv5:lwin_switch_lock_cancel     =       +level5(lwin_switch_lock_cancel)
   lv5:rwin_switch_lock_cancel     =       +level5(rwin_switch_lock_cancel)
   parens:swap_brackets   =   +parens(swap_brackets)
+  lincplus:nokpen = +lincplus(nokpen)
 
 
 ! option	=	compat
diff -r -u -N a/usr/share/X11/xkb/rules/evdev.lst b/usr/share/X11/xkb/rules/evdev.lst
--- a/usr/share/X11/xkb/rules/evdev.lst	2020-02-03 21:55:43.000000000 +0100
+++ b/usr/share/X11/xkb/rules/evdev.lst	2020-02-22 09:20:36.391589730 +0100
@@ -972,3 +972,4 @@
   solaris:sun_compat   Sun Key compatibility
   terminate            Key sequence to kill the X server
   terminate:ctrl_alt_bksp Ctrl+Alt+Backspace
+  lincplus:nokpen      Replace kpen with proper enter
diff -r -u -N a/usr/share/X11/xkb/symbols/lincplus b/usr/share/X11/xkb/symbols/lincplus
--- a/usr/share/X11/xkb/symbols/lincplus	1970-01-01 01:00:00.000000000 +0100
+++ b/usr/share/X11/xkb/symbols/lincplus	2020-02-22 09:10:38.299708564 +0100
@@ -0,0 +1,5 @@
+// Restore proper behavior of Enter key
+partial default
+xkb_symbols "nokpen" {
+    key <KPEN>	{	[	Return		]	};
+};
